emr/docker/compose.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+composeFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+
+if [ "$1" == "init" ]; then
+    docker-compose -p emr -f "$composeFilePath"/docker-compose.yml up -d
```

Comment:
The project should be `instant` not emr. This labels the containers properly and adds them to the instant network

```suggestion
    docker-compose -p instant -f "$composeFilePath"/docker-compose.yml up -d
```